### DISCUSSION TITLE
Welcome to Documentation-Template Discussions!

### DISCUSSION THREAD

**Discussion by nisabmohd** [nisabmohd/Aria-Docs#2] [General]
Sure, here's the revised message with emojis to make it more natural and welcoming:

---

## Welcome to the GitHub Documentation Template Discussion! 🎉

Hello everyone,

Welcome to the GitHub Documentation Template discussion board! We're excited to have you here and are looking forward to your contributions.

This space is dedicated to all things related to the GitHub Documentation Template. Feel free to use this discussion board for:

- **Contributing**: 🛠️ If you have improvements or new features you'd like to contribute, we'd love to see them! Please fork the repository, make your changes, and submit a pull request.
- **Raising Issues**: 🐛 Encountered a bug or something not working as expected? Please open an issue with detailed information so we can address it.
- **Feature Requests**: 💡 Have an idea for a new feature or an enhancement? Share your suggestions and let's discuss how we can make the template even better.
- **General Discussion**: 💬 Any other topics or questions related to the template are welcome here. Let's collaborate and help each other out.

We're committed to making this documentation template the best it can be, and your feedback and contributions are invaluable. Let's work together to build a great resource for everyone.

Thank you for your support and happy documenting! ✨

**Comment by crfdiab** [comment]
Can You Consider Expanding Docs logic Frontmatter with more Tags related to SEO like Featured Image, Canonical Tag, Publish Or Update Date To Be Declared In MDX files frontmatter for docs. 
Thanks In Advance

**Reply by nisabmohd** [reply]
Can you exactly describe the use case and expected features

**Reply by crfdiab** [reply]
I Mean The Ability the Declare canonical Tag in MDX frontmatter. For Example : 

```
---
title: FAQ
description: This section provides an overview of FAQ.
canonical : /docs/getting-started/faq
image   : /path-to-og-image/
alt : {alt-text-for-image}
---
```

**Comment by imranhirey** [comment]
you are just amazing

**Comment by joel-daros** [comment]
Is there any expectation of when i18n will be fully integrated into master?

**Reply by nisabmohd** [reply]
The i18n implementation for AriaDocs is done and is available in a separate branch due to feature-wise branching. Please use the i18n branch.

**Reply by nisabmohd** [reply]
https://github.com/nisabmohd/Aria-Docs/tree/i18n-support
